### ISSUE TITLE
 Feature: Adding Leftovers (For Restaurants)

### ISSUE THREAD

**Issue by sourabhkumar47** [sourabhkumar47/ResQFood#57]
Adding Leftovers (For Restaurants) to firebase

**Comment by sourabhkumar47** [comment]
> If allowed to take more than 1 issue at a time, can you elaborate on this one more?

Yes, you can take more than one issue at a time, but you should provide an update here after 4-5 days to confirm that you're still working on it; otherwise, it will be assigned to others.

**Comment by sourabhkumar47** [comment]
So are you interested in working on this issue, should I assign this to you?

**Comment by sourabhkumar47** [comment]
Should I assign this issue to you?

**Comment by ArtiGaund** [comment]
Yes, but I have little bit confusion about it. Its a form that adds the left-over food to the firebase, right? with the fields described above.

**Comment by sourabhkumar47** [comment]
> Yes, but I have little bit confusion about it. Its a form that adds the left-over food to the firebase, right? with the fields described above.

yes

**Comment by sourabhkumar47** [comment]
> om the above descripti

Use both
